templates/toil-infra-essentials.yaml

```diff
@@ -22,7 +28,7 @@ Resources:
                     - ':root'
                 - !ImportValue us-east-1-bootstrap-TravisUserArn
                 - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
-                - !If [IsSandboxAccount, "arn:aws:iam::563295687221:role/accounts-AWSIAMAdminRole-1B5HUQPC19H69", !Ref "AWS::NoValue"]
+                - !FindInMap [AdminRoleArns, !Ref "AWS::AccountId", Arn]
```

Comment:
what happened to the condition?  i think we still need it here because i believe AWS will fail if it cannot find an ARN that's been referenced in a CFN template